src/core_plugins/inspector_views/public/requests/details/req_details_response.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import {
+  EuiCodeBlock,
+} from '@elastic/eui';
+
+function RequestDetailsResponse(props) {
```

Comment:
looks pretty much the same as request one ... ? should we go with single component to remove duplication ?

Reply:
In general I would agree with you, though not sure if it's not over componentization in this place :-) Especially if we keep in mind, that we might (or might not) at some point introduce something like copying  requests to console or so, we would anyway need to separate those again. But I don't have strong feeling with regards to that, so if you prefer it way more as one component, I can change it.